src/controllers/authController.ts

```diff
@@ -0,0 +1,22 @@
+import { authRepository } from '@/repositories/authRepository.ts';
+import { Context } from 'hono';
+import { sign } from 'hono/jwt';
+
+export const authController = {
+  getToken: async (c: Context) => {
+    const { email, password } = await c.req.json();
+    const user = await authRepository.findUser(email, password);
```

Comment:
Why do you authRepository for finding a user? Please a user repository for users